assets/app/scripts/filters/resources.js

```diff
@@ -276,21 +275,16 @@ angular.module('openshiftConsole')
     };
   })
   .filter('imageRepoReference', function(){
```

Comment:
@csrwng or @liggitt can you check my changes to the `imageRepoReference` filter?